src/moscot/problems/_utils.py

```diff
@@ -48,34 +49,41 @@ def handle_joint_attr(
 
 
 def handle_cost(
-    xy: Optional[Mapping[str, Any]] = None,
-    x: Optional[Mapping[str, Any]] = None,
-    y: Optional[Mapping[str, Any]] = None,
+    xy: Mapping[str, Any] = types.MappingProxyType({}),
+    x: Mapping[str, Any] = types.MappingProxyType({}),
+    y: Mapping[str, Any] = types.MappingProxyType({}),
     cost: Optional[Union[CostFn_t, Mapping[str, CostFn_t]]] = None,
+    cost_kwargs: Union[Mapping[str, Any], Mapping[str, Mapping[str, Any]]] = types.MappingProxyType({}),
```

Comment:
Not a huge priority, but would consider adding a type alias for this and modifying to:
```
Union[Mapping[str, Any], Mapping[Literal['x', 'y', 'xy'], Mapping[str, Any]]
```
since it makes it much more obvious you can pass the kwargs to each `x/y/xy`.